src/plugins/CoreTests/src/BioStruct3DObjectTests.cpp

```diff
@@ -391,6 +393,63 @@ Task::ReportResult GTest_BioStruct3DAtomResidueName::report() {
 
     return ReportResult_Finished;
 }
+///////////////////////////////////////////////////////////////////////////////////////////
+
+void GTest_BioStruct3DMoleculeName::init(XMLTestFormat *tf, const QDomElement &el) {
+    Q_UNUSED(tf);
```

Comment:
Do not declare 'tf' name in the params list and avoid Q_UNUSED

Reply:
I can argue with that. We don't have any agreements about unused variables, actually, so, for example, I prefer wrap them with `Q_UNUSED` too

Reply:
`tf` declaration removed.